<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>startWindow</class>
 <widget class="QDialog" name="startWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RTHM- Start</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>564</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color:qlineargradient(spread:pad, x1:0.005, y1:0, x2:1, y2:1, stop:0 rgba(200, 197, 191, 255), stop:1 rgba(255, 255, 255, 255))}</string>
   </property>
   <widget class="QLabel" name="imgStartScreenLogo">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>30</y>
      <width>521</width>
      <height>391</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/startScreenLogo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>397</x>
      <y>430</y>
      <width>231</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;START THE APP&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;
border-radius:5px;
background-color: rgb(255, 102, 0);
padding: 4px;
box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;
font: 24pt &quot;Montserrat&quot;;
font-weight: bold;
</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInfo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>240</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: left;
border: none;
background-color: #FFE2CE;
font: 8pt &quot;Montserrat&quot;;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>RTHM-DEVICE- V.01.03.22 BETA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInfo_2">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/icons/i.png</normaloff>images/icons/i.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images/startLogo.qrc"/>
 </resources>
 <connections/>
</ui>
